2019-07-26 11:47:09 INFO:Workflow_1_2_2.py:28 Setting Workflow basic metadata 

2019-07-26 11:47:09 INFO:Workflow_1_2_2.py:89 Workflow initialization 

2019-07-26 11:47:09 DEBUG:PyroUtil.py:176 Can connect to a LISTENING port of nameserver on 172.30.0.1:9090 

2019-07-26 11:47:09 DEBUG:PyroUtil.py:188 Connected to NameServer on 172.30.0.1:9090. Pyro4 version on your localhost is 4.75 

2019-07-26 11:47:09 INFO:PyroUtil.py:556 Located Job Manager eX_DigimatMF_JobManager at: 172.30.0.112 44382 None None 

2019-07-26 11:47:09 DEBUG:PyroUtil.py:252 Application eX_DigimatMF_JobManager, found URI PYRO:obj_81caf4362ac24beebc002ed4a250e246@172.30.0.112:44382 on ('172.30.0.112', 44382, 'None', 'None') from a nameServer <Pyro4.core.Proxy at 0x7fee546d0eb8; connected IPv4; for PYRO:Pyro.NameServer@172.30.0.1:9090> 

2019-07-26 11:47:09 INFO:PyroUtil.py:260 Connecting to application eX_DigimatMF_JobManager with <Pyro4.core.Proxy at 0x7fee546e3390; not connected; for PYRO:obj_81caf4362ac24beebc002ed4a250e246@172.30.0.112:44382> 

2019-07-26 11:47:09 DEBUG:PyroUtil.py:262 Connected to Mupif.JobManager.SimpleJobManager2 with the application eX_DigimatMF_JobManager 

2019-07-26 11:47:09 INFO:PyroUtil.py:556 Located Job Manager MUL2.JobManager@UseCase1 at: 172.30.0.172 44388 None None 

2019-07-26 11:47:09 DEBUG:PyroUtil.py:252 Application MUL2.JobManager@UseCase1, found URI PYRO:obj_5e4d670179364a2cacbba0fbac3a5b2b@172.30.0.172:44388 on ('172.30.0.172', 44388, 'None', 'None') from a nameServer <Pyro4.core.Proxy at 0x7fee546d0eb8; connected IPv4; for PYRO:Pyro.NameServer@172.30.0.1:9090> 

2019-07-26 11:47:09 INFO:PyroUtil.py:260 Connecting to application MUL2.JobManager@UseCase1 with <Pyro4.core.Proxy at 0x7fee546e30f0; not connected; for PYRO:obj_5e4d670179364a2cacbba0fbac3a5b2b@172.30.0.172:44388> 

2019-07-26 11:47:09 DEBUG:PyroUtil.py:262 Connected to Mupif.JobManager.SimpleJobManager2 with the application MUL2.JobManager@UseCase1 

2019-07-26 11:47:09 INFO:Workflow_1_2_2.py:100 Created lammps job 

2019-07-26 11:47:09 DEBUG:PyroUtil.py:597 Trying to connect to JobManager 

2019-07-26 11:47:10 INFO:PyroUtil.py:615 Allocated job, returned record from jobManager:(1, '24@eX_DigimatMF_JobManager', 9023) 

2019-07-26 11:47:10 INFO:PyroUtil.py:638 Scenario: Connecting to 24@eX_DigimatMF_JobManager 9023 

2019-07-26 11:47:10 DEBUG:PyroUtil.py:252 Application 24@eX_DigimatMF_JobManager, found URI PYRO:obj_05a44a230f3c4a85a716af7ed361fe7b@172.30.0.112:9023 on ('172.30.0.112', 9023, 'None', 'None') from a nameServer <Pyro4.core.Proxy at 0x7fee546d0eb8; connected IPv4; for PYRO:Pyro.NameServer@172.30.0.1:9090> 

2019-07-26 11:47:10 INFO:PyroUtil.py:260 Connecting to application 24@eX_DigimatMF_JobManager with <Pyro4.core.Proxy at 0x7fee546f7438; not connected; for PYRO:obj_05a44a230f3c4a85a716af7ed361fe7b@172.30.0.112:9023> 

2019-07-26 11:47:11 DEBUG:PyroUtil.py:262 Connected to DigimatApplication with the application 24@eX_DigimatMF_JobManager 

2019-07-26 11:47:11 INFO:Workflow_1_2_2.py:102 Created digimat job 

2019-07-26 11:47:11 DEBUG:PyroUtil.py:597 Trying to connect to JobManager 

2019-07-26 11:47:12 INFO:PyroUtil.py:615 Allocated job, returned record from jobManager:(1, '24@MUL2.JobManager@UseCase1', 8920) 

2019-07-26 11:47:12 INFO:PyroUtil.py:638 Scenario: Connecting to 24@MUL2.JobManager@UseCase1 8920 

2019-07-26 11:47:12 DEBUG:PyroUtil.py:252 Application 24@MUL2.JobManager@UseCase1, found URI PYRO:obj_d7c524028479434eb4ef720749daec4c@172.30.0.172:8920 on ('172.30.0.172', 8920, 'None', 'None') from a nameServer <Pyro4.core.Proxy at 0x7fee546d0eb8; connected IPv4; for PYRO:Pyro.NameServer@172.30.0.1:9090> 

2019-07-26 11:47:12 INFO:PyroUtil.py:260 Connecting to application 24@MUL2.JobManager@UseCase1 with <Pyro4.core.Proxy at 0x7fee546f7748; not connected; for PYRO:obj_d7c524028479434eb4ef720749daec4c@172.30.0.172:8920> 

2019-07-26 11:47:12 DEBUG:PyroUtil.py:262 Connected to MUL2_FEM with the application 24@MUL2.JobManager@UseCase1 

2019-07-26 11:47:12 INFO:Workflow_1_2_2.py:104 Created mul2 job 

2019-07-26 11:47:12 INFO:Workflow_1_2_2.py:110 Working digimat solver on server DigimatApplication 

2019-07-26 11:47:12 INFO:Workflow_1_2_2.py:112 Working mul2 solver on server MUL2_FEM 

2019-07-26 11:47:12 INFO:Workflow_1_2_2.py:117 Metadata were successfully validate 

2019-07-26 11:47:12 INFO:Workflow_1_2_2.py:128 Setting Execution Metadata of Lammps 

2019-07-26 11:47:12 INFO:Workflow_1_2_2.py:129 Setting Execution Metadata of Digimat 

2019-07-26 11:47:12 INFO:Workflow_1_2_2.py:130 Setting Execution Metadata of Mul2 

2019-07-26 11:47:12 DEBUG:Workflow.py:131 Step 1: t=1 dt=1 

2019-07-26 11:47:13 INFO:Workflow_1_2_2.py:174 Running Lammps 

2019-07-26 11:47:13 INFO:Workflow_1_2_2.py:176 Running Digimat 

2019-07-26 11:47:15 INFO:Workflow_1_2_2.py:237 Running Mul2 

2019-07-26 11:48:10 INFO:Model.py:558 RemoteApplication: Terminating jobManager job 24@eX_DigimatMF_JobManager on eX_DigimatMF_JobManager 

2019-07-26 11:48:10 INFO:Model.py:558 RemoteApplication: Terminating jobManager job 24@MUL2.JobManager@UseCase1 on MUL2.JobManager@UseCase1 

2019-07-26 11:48:11 INFO:Model.py:496 Terminating model 

2019-07-26 11:48:11 INFO:Workflow_1_2_2.py:333 Requested KPI : Buckling Load: 10001.256 N 

2019-07-26 11:48:11 INFO:Model.py:496 Terminating model 

2019-07-26 11:48:11 INFO:Workflow_1_2_2.py:335 Process complete 

